src/blueprint/definitions.rs

```diff
@@ -226,25 +224,21 @@ fn process_path(context: ProcessPathContext) -> Valid<Type, String> {
     Valid::succeed(to_type(field, Some(is_required)))
 }
 
-fn to_enum_type_definition(
-    name: &str,
-    type_: &config::Type,
-    variants: &BTreeSet<String>,
-) -> Valid<Definition, String> {
-    let enum_type_definition = Definition::Enum(EnumTypeDefinition {
-        name: name.to_string(),
+fn to_enum_type_definition((name, variants): (&String, &Enum)) -> Definition {
+    Definition::Enum(EnumTypeDefinition {
+        name: name.to_owned(),
         directives: Vec::new(),
-        description: type_.doc.clone(),
+        description: None,
```

Comment:
We should have the ability to set descriptions. 
NOTE: Please also add snapshot tests to ensure descriptions are added.